app/src/org/commcare/dalvik/activities/FormRecordListActivity.java

```diff
@@ -93,7 +96,8 @@
     private ImageButton barcodeButton;
     private Spinner filterSelect;
     private ListView listView;
-    
+    private SearchView searchView;
```

Comment:
The linter says that this variable can be made local to the onCreateOptionsMenu method.